debian/clearwater-auto-config-generic.init.d

```diff
@@ -63,6 +63,7 @@ do_auto_config()
 
   sed -e 's/^local_ip=.*$/local_ip='$ip'/g
           s/^public_ip=.*$/public_ip='$ip'/g
+          s/^etcd_cluster=.*$/etcd_cluster='$ip'/g
```

Comment:
AWS has a local/public IP distinction - this isn't relevant in auto-config-generic, where we just use the first IP address of the box.